packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -162,6 +172,28 @@ export default class QuanticGeneratedAnswer extends LightningElement {
   updateState() {
     this.state = this.generatedAnswer.state;
     this.updateFeedbackState();
+    console.log(this.getGeneratedAnswerStatus());
+    this.ariaLiveMessage.dispatchMessage(this.getGeneratedAnswerStatus());
+  }
+
+  getGeneratedAnswerStatus() {
+    if (!this.state.isVisible) {
+      return this.labels.generatedAnswerIshidden;
+    }
+    const isGenerating = this.state.isLoading || this.state.isStreaming;
+    const hasAnswer = !!this.state.answer;
+    const hasError = this.hasRetryableError;
+
+    if (hasError) {
+      return this.labels.couldNotGenerateAnAnswer;
+    }
+    if (isGenerating) {
+      return this.labels.generatingAnswer;
+    }
+
+    return hasAnswer
+      ? I18nUtils.format(this.labels.answerGenerated, this.answer)
+      : '';
   }
```

Comment:
```suggestion
  getGeneratedAnswerStatus() {
    if (!this.state.isVisible) {
      return this.labels.generatedAnswerIshidden;
    }

    if (this.hasRetryableError) {
      return this.labels.couldNotGenerateAnAnswer;
    }
    
    const isGenerating = this.state.isLoading || this.state.isStreaming;
    if (isGenerating) {
      return this.labels.generatingAnswer;
    }
    
    const hasAnswer = !!this.state.answer;
    return hasAnswer
      ? I18nUtils.format(this.labels.answerGenerated, this.answer)
      : '';
  }
```

Small clarity improvement IMO